server/src/test/resources/configs/with-secret-size-limit-123456.yaml

```diff
@@ -0,0 +1,159 @@
+maximumSecretSizeInBytes: 123456
+
+server:
```

Comment:
Front and center.

Unfortunately the heavy validation means that a lot of unrelated values need to be specified in order to have a valid config. One alternative would be to disable validation in the tests but that wouldn't match Keywhiz's actual behavior.

Reply:
Makes sense, not pretty to look at but at least we're testing keywhiz' actual behavior.

Reply:
Though curious: since we have two separate yamls here (one with the secret size, one without, could we have a "common" yaml (that isn't actually used in any of the tests) and `with-secret-size-limit-123456.yaml` could then have an `"%include "test-common.yaml"`?

Reply:
Huh, I've seen the include syntax but it hadn't occurred to me to try it. I don't know how it works under the hood but my guess is that it'll fail for this use case because the test files are resources packaged up in JARs and not actual files on the filesystem like they are in deployed apps.

Reply:
I tested this out locally and, as expected, include directives do not work for classpath resources.